examples/basic/annotations/legends_item_visibility.py

```diff
@@ -1,3 +1,11 @@
+''' Marker and line plots that demonstrate manual control of legend visibility of individual items
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.plotting.figure.line
+    :refs: :ref:`ug_basic_annotations`
```

Comment:
I think we have a section specifically about legend item visibility (which is actually using this example), so you could do this:

```suggestion
    :refs: :ref:`ug_basic_annotations` > :ref:`ug_basic_annotations_legends_item_visibility`
```